ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -23,12 +23,24 @@ parameters:
 - name: DEPLOY_VERSION
   value: "v3.11"
 - name: TEST_IMAGE
-  value: "registry.svc.ci.openshift.org/azure/test-base:latest"
+  value: quay.io/kwoodson/bushslicer
+  #value: "registry.svc.ci.openshift.org/azure/test-base:latest"
 - name: TEST_COMMAND
-  value: "make e2e"
+  value: "/usr/bin/scl enable rh-git29 rh-ror50 -- cucumber features/cli/build.feature:5"
+  #value: "make e2e"
 - name: RESOURCEGROUP_SUFFIX
   generate: expression
   from: "[a-z]{8}"
+- name: TESTENV
+  value: BUSHSLICER_V3
```

Comment:
You should be setting these values in the bushslicer prowjob

Reply:
you mean jsut define `- name: TESTENV` here, without values and pass those from prow?

Reply:
Yes, the _values_ need to come from the bushslicer prowjob.

Reply:
Will move to the periodic job.